magda-web-client/src/Components/Dataset/Add/DatasetAddMetadataPage.tsx

```diff
@@ -563,6 +323,7 @@ class NewDataset extends React.Component<Props, State> {
                     orgUnitOwnerId: dataset.owningOrgUnitId,
                     custodianOrgUnitId: dataset.custodianOrgUnitId
                 },
+                currency,
```

Comment:
This needs to have the same logic as on `derivedFrom` below for `supersededBy` - otherwise dereferencing doesn't work and there's extra info put in:

<img width="436" alt="image" src="https://user-images.githubusercontent.com/900555/64756420-3abd8e00-d572-11e9-8cda-5d6c0dcb0d3d.png">

Reply:
Ah~ Thanks 👍 Will updated accordingly~